src/controllers/office.js

```diff
@@ -0,0 +1,55 @@
+/* eslint-disable max-len */
+import Office from '../models/office';
+import ValidateOffice from '../middlewares/helperoffice';
+
+const ControllerOffice = {
+  createOffice(req, res) {
+    // eslint-disable-next-line no-restricted-globals
+    if (isNaN(req.body.age)) {
+      return res.status(406).json({
+        status: 406,
```

Comment:
Error message should be more descriptive. What must be a number? message should say something like `Age Limit entered must be a number`